book/src/user/docker.md

```diff
@@ -18,27 +18,12 @@ docker run --detach zfnd/zebra:1.0.0-rc.8
 
 ```shell
 git clone --depth 1 --branch v1.0.0-rc.8 https://github.com/ZcashFoundation/zebra.git
-docker build --file docker/Dockerfile --target runtime --tag zebra:local
+docker build --file docker/Dockerfile --target runtime --tag zebra:local .
 docker run --detach zebra:local
 ```
 
-### Alternatives
-
-See the Zebra [build instructions](https://github.com/ZcashFoundation/zebra#build-instructions).
```

Comment:
I would keep this section, but referencing the actual link in the README https://github.com/ZcashFoundation/zebra#building-zebra

Reply:
Done in [c8fe01f](https://github.com/ZcashFoundation/zebra/pull/6723/commits/c8fe01fd467c376f988121dade294639aff6257e).